### PR TITLE
743 Add printing to both apis

### PR DESCRIPTION
**Pull request by andreievg**
#743.3

Should be pretty self explanatory. Wanted to make native functionality can be added (this of course lack all of the work Mark did on printing, but can be easily ported and extended, and I think allow for full control of native).

Both Electron and Android will just open print preview in this case.

### CODE REVIEW

File: client/packages/electron/src/nativePrint.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { BrowserWindow } from 'electron';
+
+export const nativePrint = async (html: string) => {
```
Comment (andreievg) [inline client/packages/electron/src/nativePrint.ts:3]: Converting to pdf seems like the only way to do this, i couldn't figure out another way 🤷

Comment (mark-prins) [inline client/packages/electron/src/nativePrint.ts:3]: I'm curious as to why the change? the move to native printing in electron removes the print preview, so the overall user experience is worse. I'm not sure what the benefit is..?

**Review by mark-prins — COMMENTED** [review]
the android printing gives me 12 pages for an empty stocktake - so something is going strange!

print_1
print_2
print_3

and overall.. I'm missing the point! not sure why the native printing is required?

**Comment by mark-prins** [comment]
side note: had to upgrade rust to get the server to compile! If you have trouble running, `rustup update`

**Comment by andreievg** [comment]
In the next PR printing was kept and seems to be working well, closing this one, but would like to make an example NativeApi extension for the `info` icon for sync errors
